SYCL/Basic/kernel_program_cache.cpp

```diff
@@ -0,0 +1,43 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env CreateMultipleRootDevices=2 NEOReadDebugKeys=1 \
+// RUN: SYCL_DEVICE_FILTER="level_zero:gpu" %t.out
+// RUN: env CreateMultipleRootDevices=2 NEOReadDebugKeys=1 \
+// RUN: SYCL_DEVICE_FILTER="opencl:gpu" %t.out
```

Comment:
```suggestion
// RUN: env CreateMultipleRootDevices=2 NEOReadDebugKeys=1  %GPU_RUN_PLACEHOLDER %t.out
// REQUIRES: (opencl || level_zero) && gpu
```
We have separate runs for level_zero and opencl.

Reply:
Changed it